mesop/components/audio/audio.py

```diff
@@ -4,21 +4,18 @@
 
 @register_native_component
 def audio(
-  *,
-  src: str | None = None,
-  key: str | None = None,
+  *, src: str | None = None, key: str | None = None, autoplay: bool = False
 ):
   """
   Creates an audio component.
 
   Args:
       src: The URL of the audio to be played.
+      autoplay: boolean value indicating if the audio should be autoplayed or not.
```

Comment:
Add note here about the subtle issue with page interaction so that it shows up in the API docs.